pysparkle/tests/unit/data_quality/test_data_quality_utils.py

```diff
@@ -0,0 +1,74 @@
+from unittest.mock import patch
+from pathlib import Path
+import shutil
+
+import pytest
+
+from pysparkle.data_quality.data_quality_utils import *
+
+TEST_GX_DIR = Path(__file__).parent.resolve() / "great_expectations"
+
+TEST_DQ_CONF = {
+    "container_name": "staging",
+    "datasource_name": "movies_metadata",
+    "expectation_suite_name": "movies_metadata_suite",
+    "gx_directory_path": TEST_GX_DIR,
+    "validation_config": [
+        {
+            "column_name": "adult",
+            "expectations": [
+                {
+                    "expectation_type": "expect_column_values_to_not_be_null",
+                    "expectation_kwargs": {},
+                }
+            ],
+        }
+    ],
+}
+
+
+def test_create_datasource_context():
+    context = create_datasource_context(TEST_DQ_CONF["datasource_name"], TEST_DQ_CONF["gx_directory_path"])
```

Comment:
I would rather have a fixture with creating a datasource_context that would create it using pytest's `tmp_path` fixture. That way we wouldn't have to bother with removing this directory. Right now, if the test fails, it won't be removed. But I'm OK to do it in the next round. If you agree, could you add a task to one of the DQ stories?

Reply:
I've got another PR going in soon anyway I'll do it in that